modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -674,9 +694,33 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
      */
     function toJSON()
     {
+        $db   = Database::singleton();
+        $user = User::singleton();
+        $ids  = $db->pselect(
```

Comment:
This request is done twice. It should be in a function like `getCandidateIdsMapping` or something like that. Or put in a class variable so it can be accessed in toJSON() and _setFilterForm()